### PR TITLE
WAF: Save log info in the database

### PR DESCRIPTION
**Pull request by miguelxpn**
#### Changes proposed in this Pull Request:

* Fix db logging when a request is blocked by our WAF

#### Jetpack product discussion

p1650845956991089/1650639283.564489-slack-C029WFNV69M

#### Does this pull request change what data or activity we track or use?

No.
#### Testing instructions:

* Checkout this branch and build the plugin with all it's dependencies
* Force a request to be blocked with the `jetpack-waf-always-block` query string parameter
* Verify that the request was logged to the blocklog file and the database as well.
*

### CODE REVIEW

File: projects/packages/waf/src/class-waf-runtime.php [hunk 1]
```diff
@@ -299,6 +299,9 @@ private function write_blocklog_row( $log_data ) {
 		$statement = $conn->prepare( "INSERT INTO {$table_prefix}jetpack_waf_blocklog(reason,rule_id, timestamp) VALUES (?, ?, ?)" );
 		if ( false !== $statement ) {
 			$statement->bind_param( 'sis', $log_data['reason'], $log_data['rule_id'], $log_data['timestamp'] );
+			$statement->execute();
+
+			var_dump( $conn );
```
Comment (ArSn) [inline projects/packages/waf/src/class-waf-runtime.php:304]: I don't think we want that in here? :) 
```suggestion
```

Comment (miguelxpn) [reply projects/packages/waf/src/class-waf-runtime.php]: Oops! Silly me, thanks!

**Review by ArSn — DISMISSED** [review]

**Review by singerb — APPROVED** [review]
Looks good from our side.

**Review by singerb — APPROVED** [review]
Still looks good, now with fewer random warnings!

**Pull request merged**